packages/php/blueprint/src/Exporters/ExportInstallPluginSteps.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Automattic\WooCommerce\Blueprint\Exporters;
+
+use Automattic\WooCommerce\Blueprint\Steps\InstallPlugin;
+use Automattic\WooCommerce\Blueprint\UseWPFunctions;
+
+class ExportInstallPluginSteps implements StepExporter {
+	use UseWPFunctions;
+	private bool $include_private_plugins = false;
+
+	public function include_private_plugins( bool $boolean ) {
+		$this->include_private_plugins = $boolean;
+	}
+
+	public function export() {
+		$plugins = $this->wp_get_plugins();
+
+		// @todo temporary fix for JN site -- it includes WooCommerce as a custom plugin
+		// since JN sites are using a different slug.
+		$exclude = array( 'WooCommerce', 'WooCommerce Beta Tester' );
+		$steps   = array();
+		foreach ( $plugins as $path => $plugin ) {
+			if ( in_array( $plugin['Name'], $exclude, true ) ) {
+				continue;
+			}
+			// skip inactive plugins for now.
+			if ( ! $this->wp_is_plugin_active( $path ) ) {
+				continue;
+			}
+			$slug = dirname( $path );
```

Comment:
We assume that the directory name is a slug, but users may install plugins with different directory names, like our live branch feature. I'm not sure if there is a better way to do this, though. 🤔